backend/app/tests/test_api_security.py

```
@@ -9,30 +9,39 @@
 import pytest
 import json
 import os
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)